File: BackEnd/src/main/java/com/minhtuan/commercemanager/message/response/OrderResponse.java
package com.minhtuan.commercemanager.message.response;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Getter;
import lombok.Setter;
import org.springframework.format.annotation.DateTimeFormat;

import java.util.Date;

@Getter
@Setter
public class OrderResponse {
    private String orderId;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd", timezone = "Asia/Ho_Chi_Minh")
    private Date dateOfOrder;
    private String firstnameOfReveiver;
    private String lastnameOfReveiver;
    private String addressOfReceiver;
    private String phoneOfReceiver;
    private Integer status;
    private Integer payment;
}

File: FrontEnd/src/app/response/order-response.ts
import { Employee } from "../model/employee";

export class OrderResponse {
    orderId!: string;
    dateOfOrder!: string;
    firstnameOfReveiver!: string;
    lastnameOfReveiver!: string;
    addressOfReceiver!: string;
    phoneOfReceiver!: string;
    status!: number;
    deliveryDate!: string;
    employee!: Employee;
    employeeDelivery!: Employee;
    payment!: number;
}
